README.md

```diff
@@ -1,66 +1,60 @@
-
- - **Kernel 6.0+:**
-   > Keyboard works without patching
-
- - _**For more information take a look at the [issues](https://github.com/tomsom/yoga-linux/issues)**_
-
-## Problems:
- - [x] **Keyboard not working**
-   > The keyboard doesn't work out of the box (Fixed in kernel 6.0)
-
- - [x] **Internal mic not working**
-   > Internal mic does not work (Fixed in kernel 6.1)
-
- - [x] **S0 sleep (s2idle)**
-   > A lower power sleep state: device wakes up almost immediately, but often breaks the keyboard or touchscreen (Fixed in kernel 6.1)
-
- - [x] **The speakers suck**
-   > Switch to the bottom firing woofers makes the sound bearable, having all 4 speakers working in parallel is still WIP
-
- - [x] **S4 sleep (hibernating) (guide is still missing)**
-   > As an alternative for S3 sleep S4 sleep can be made use of (requires a swap partition)
-   
- - [x] **Screen rotation**
-   > Automatic rotation does not work out of the box in GNOME for some reason. Can be fixed with an [extension](https://github.com/shyzus/gnome-shell-extension-screen-autorotate).
-
- - [ ] **S3 sleep (suspending)**
-   > No option for regular S3 sleep support; only for Microsofts S0ix sleep which is not supported by Linux, at least for AMD Processors
-
- - [ ] **GRUB scaling**
-   > Because of the 2.8K display the text in GRUB is very small and hardly readable
-   > 
-   > This can be partially fixed by using a GRUB theme that forces a font size, or manually configuring GRUB
-
-## Other issues:
- - These issues don't seem to impact all Yoga 7 Gen 7 users
- - **Cursor jumps around the screen**
-   - This is caused by PSR (Panel Self-Refresh), and can be worked around using `amdgpu.dcdebugmask=0x10` as a kernel parameter
-   - This value is documented [here](https://github.com/torvalds/linux/blob/8813381a62e1f1703f8fbeccc5fa4fcc988be882/drivers/gpu/drm/amd/include/amd_shared.h#L250)
-
- - **System crashes after resuming**
-   - Yoga laptops with Samsung SSDs may become unstable after resuming from s2idle
-   - This is a firmware bug in the SSD (PCI ID `0xa80b`)
-   - This issue is better documented [here](https://github.com/tomsom/yoga-linux/issues/9)
-
-## Other stuff:
-### More issues I've noticed:
- - ~~Kernel 6.1 is incompatible with my (Delock) USB-C dock(system freezes)~~(Fixed in kernel 6.1-rc7)
- - Waking up from hibernation often needs longer than a reboot/regular system start
-
+## Latest Fixes:
+> **Kernel independent:**
+ > - [x] **S4 sleep | Hibernating (guide is still missing)**
+ > - As an alternative for S3 sleep, S4 sleep can be made use of (requires a SWAP partition).
+ > - [x] **Screen rotation**
+ > - Automatic rotation does not work out of the box in GNOME for some reason.
+ > - Can be fixed with an [extension](https://github.com/shyzus/gnome-shell-extension-screen-autorotate).
+ 
+> **Kernel 6.1+:**
+ > - [x] **Fixed incompatibility with 'Delock' USB-C dock**
```

Comment:
This is not yet fixed, see #13

Reply:
Will likely only get properly fixed in 6.2, some docks can still work under 6.1.